README.md

```diff
@@ -1 +1,122 @@
-# Dash Template
+### Interested in contributing a code example?
+
+Please take a look at our [contribution guidelines](CONTRIBUTING.md) before getting started.
+Thank you!
+
+The Dash template is intended for demos that would benefit from a user interface. This user
+interface could include settings to run and customize the problem, an interactive graphical element,
+or tables/charts to compare different solutions. This template is also useful for demos that are
+intended for a general audience, as it is more approachable for those without a technical background.
+
+<!-- Before submitting your code, please delete everything above and including this comment. -->
+<!-- The following is a README template for your new demo. -->
+
+# Demo Name
+
+Describe your demo and specify what it is demonstrating. Consider the
+following questions:
+
+* Is it a canonical problem or a real-world application?
+* Does it belong to a particular domain such as material simulation or logistics?
+* What level of Ocean proficiency does it target: beginner, advanced?
+
+A clear description allows us to properly categorize your demo.
+
+Please include a screenshot of your demo below.
+
+![Demo Example](static/demo.png)
+
+<!-- Below is boilerplate instructions to be included, as is, in the final demo. -->
+
+## Installation
+You can run this example without installation in cloud-based IDEs that support the
+[Development Containers specification](https://containers.dev/supporting) (aka "devcontainers").
```

Comment:
I wonder if this can be rewritten to include that we're usually using codespaces for this. I'm not sure most users would be interested in the details here, but rather just would like to be able to use the run-in-codepaces button that we usually add at the top of the demo readme.

Reply:
Not sure if this is what you meant or not so let me know: https://github.com/dwave-examples/template-dash/pull/3/commits/6d604fbcbc6a902583e54bfcb8a200ca1a54ee27#:~:text=(aka%20%22devcontainers%22).-,%5BDevelopment%20Containers%20specification%5D(https%3A//containers.dev/supporting),such%20as%20GitHub%20Codespaces.,-For%20development%20environments

Reply:
I think this suffices for now. We might want to add in the codespaces badge later if and when an example gets accepted.